src/Illuminate/Testing/ParallelRunner.php

```diff
@@ -126,6 +126,8 @@ protected function forEachProcess($callback)
      * Creates the application.
      *
      * @return \Illuminate\Contracts\Foundation\Application
+     *
+     * @throws \RuntimeException
```

Comment:
RuntimeException and Error don't usually show in the throws doc. One normally assumes they can always be thrown by any code.

Reply:
I see. There are already 60+. So should I delete them (that I've added) or leave them as they are?

Reply:
Just don't add any more unnecessary ones.